src/app/components/feed/feeds.factory.js

```diff
@@ -293,6 +293,30 @@
       };
 
       /**
+       * notifying changes to the remote peers.
```

Comment:
updateDisplay basically sends the status as soon as OK is clicked in the UI, making it as updateLocalPic did not do the trick as there was nothing to update name(display attribute) on the peers side, thus added getDisplay and setDisplay and added display to setStatus to make it work. Please review and correct if this method does not follow jangouts practices.